docs/app/Examples/views/Statistic/Types/Groups.js

```diff
@@ -2,13 +2,14 @@ import React from 'react'
 import { Statistic } from 'stardust'
 
 const { Group, Label, Value } = Statistic
+
 const items = [
-  { label: 'Faves', value: 22 },
+  { label: 'Faves', value: '22' },
   { label: 'Views', value: '31,200' },
-  { label: 'Members', value: 22 },
+  { label: 'Members', value: '22' },
```

Comment:
It cannot be `node`, as it's used for key on `StepGroup`.

----

```js
Proptypes.oneOf([Proptypes.number, Proptypes.string])
```

I think that it's not good solution. This raises the question of the behavior of other properties.